packages/dev/core/src/Gizmos/gizmo.ts

```diff
@@ -49,6 +49,14 @@ export enum GizmoAnchorPoint {
     Pivot,
 }
 
+/**
+ * Coordinate system mode: Local or World. Defines how axis is aligned: either on world axis or transform local axis
+ */
+export enum GizmoCoordinates {

```

Comment:
nit: I would have called this `GizmoCoordinateMode` (or `GizmoCoordinateSystemMode)`, because coordinates sound like coordinates of a point

Reply:
Fixed.